k8s/tools/admin-node/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
+
+RUN apt-get update\
+        && apt-get install -y --no-install-recommends mysql-client python-setuptools python-pip openssl\
```

Comment:
Nit: since we're using the `python:3` image, `python-setuptools` and `python-pip` can be removed from the install here.